object/product.go

```diff
@@ -245,6 +246,14 @@ func BuyProduct(id string, user *User, providerName, pricingName, planName, host
 	if err != nil {
 		return nil, nil, err
 	}
+
+	var successRedirectUrl string
+	if product.SuccessUrl != "" {
```

Comment:
Why not use successRedirectUrl := product.SuccessUrl ? or directly use product.SuccessUrl?